node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -1027,6 +1027,25 @@ impl Initialized {
 			}
 		}
 
+		// Notify ChainSelection if a dispute has concluded against a candidate. ChainSelection
+		// will need to mark the candidate's relay parent as reverted.
+		if import_result.is_freshly_concluded_against() {
+			if let Some(info) = self.scraper.get_included_candidate_info(candidate_hash) {
+				ctx.send_message(ChainSelectionMessage::DisputeConcludedAgainst(
```

Comment:
That makes sense! Though I'd offer one modification. 

From the perspective of `propagate_viability_update` in chain selection we aren't reverting to any particular block. 

The function takes as its parameter a block to be marked as `explicitly_reverted` and assumes that via the propagation process we end up with a new active leaf, the "reverted to" block.

So perhaps the name `ChainSelectionMessage::RevertBlock` instead?

Reply:
Better yet, since one disputed candidate can cause the reversion of multiple relay blocks on different forks we can go with `ChainSelectionMessage::RevertBlocks`